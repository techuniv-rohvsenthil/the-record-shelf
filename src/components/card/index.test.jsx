import React from 'react';
import { render } from '@testing-library/react';
import Card from './index';


describe('the Card component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Card text="Test text" img="img-src" alt="img-alt"> </Card>);

    expect(asFragment()).toMatchSnapshot();
  });
});
