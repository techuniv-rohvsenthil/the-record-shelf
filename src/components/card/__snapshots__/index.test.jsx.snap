// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Card component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <img
      alt="img-alt"
      height="100px"
      src="img-src"
      width="100px"
    />
    <button
      data-testid="card-btn"
      type="button"
    >
      Test text
    </button>
  </div>
</DocumentFragment>
`;
